packages/react-native/package.json

```diff
@@ -1,36 +1,46 @@
 {
-    "name": "@honeybadger-io/react-native",
-    "version": "5.2.0",
-    "description": "Honeybadger.io for React Native.",
-    "author": "Andrey Butov <andreybutov@antair.com> (https://andreybutov.com)",
-    "license": "MIT",
-    "types": "index.d.ts",
-    "main": "index.js",
-    "scripts": {
-        "build": "tsc",
-        "test": "echo \"Error: no test specified\""
-    },
-    "bin": {
-        "honeybadger-generate-sourcemaps": "./honeybadger-generate-sourcemaps.sh"
-    },
-    "repository": {
-        "type": "git",
-        "url": "git@github.com:honeybadger-io/honeybadger-js.git"
-    },
-    "keywords": [
-        "Honeybadger"
-    ],
-    "homepage": "https://github.com/honeybadger-io/honeybadger-js/tree/master/packages/react-native",
-    "peerDependencies": {
-        "react": ">=16.13.1",
-        "react-native": ">=0.60.0-rc.0 <1.0.x"
-    },
-    "devDependencies": {
-        "react": "^18.2.0",
-        "react-native": "^0.71.3",
-        "typescript": "^4.3.5"
-    },
-    "engines": {
-        "node": ">=14"
-    }
+  "name": "@honeybadger-io/react-native",
+  "version": "5.2.0",
+  "description": "Honeybadger.io for React Native.",
+  "author": "Andrey Butov <andreybutov@antair.com> (https://andreybutov.com)",
+  "license": "MIT",
+  "types": "dist/index.d.ts",
+  "main": "dist/index.js",
+  "scripts": {
+    "build": "rollup -c",
+    "build:clean": "rm -rf dist && npm run build",
+    "test": "echo \"Error: no test specified\""
+  },
+  "bin": {
+    "honeybadger-generate-sourcemaps": "./honeybadger-generate-sourcemaps.sh"
```

Comment:
I'm still not sure how npm works with all of this being moved because of the monorepo stuff. Also, how is the "generate-sourcemaps" script in bin supposed to work now? There's lots of stuff that I'm not sure about anymore :) But I leave the npm stuff to the experts.

Reply:
> Also, how is the "generate-sourcemaps" script in bin supposed to work now?

It should work in the same way, no change in behavior.